Cargo.toml

```diff
@@ -35,7 +35,9 @@ targets = ["aarch64-apple-ios", "aarch64-linux-android", "x86_64-apple-darwin",
 features = ["all"]
 
 [target.'cfg(any(unix, target_os = "wasi"))'.dependencies]
-libc = { git = "https://github.com/wasix-org/libc.git" }
```

Comment:
I reckon we should directly specify the `wasm32-wasmer-wasi` target here. 

That way it would actually be mergable as is, and Thomas can decide if we need to wait for libc.

But would be great to get this merged first to have some motivators for libc/tokio.

Reply:
Resolved by publishing `wasix-libc` and using that for `wasm32-wasmer-wasi`.